contrib/mesos/pkg/scheduler/service/service.go

```diff
@@ -351,7 +361,7 @@ func (s *SchedulerServer) serveFrameworkArtifactWithFilename(path string, filena
 }
 
 func (s *SchedulerServer) prepareExecutorInfo(hks hyperkube.Interface) (*mesos.ExecutorInfo, error) {
-	ci := &mesos.CommandInfo{
+		ci := &mesos.CommandInfo{
```

Comment:
@linsun: I am referring specifically to http://kubernetes.io/v1.1/docs/devel/development.html#hooks